frontend/src/v5/ui/routes/viewer/tickets/ticketGroupsCard/groups/groups.styles.ts

```diff
@@ -0,0 +1,143 @@
+/**
+ *  Copyright (C) 2023 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { Typography } from '@controls/typography';
+import styled, { css } from 'styled-components';
+import { Accordion } from '@controls/accordion/accordion.component';
+import { GroupIcon } from '../../../groups/groupItem/groupIcon/groupIcon.styles';
+
+const GroupStyling = css`
+	border: 0;
+	border-radius: 8px;
+	height: 42px;
+	box-sizing: border-box;
+	background-color: ${({ theme }) => theme.palette.primary.contrast};
+	padding: 7px;
+	display: flex;
+	align-items: center;
+`;
+
+export const GroupItemButtons = styled.div`
+	width: 100%;
+	display: flex;
+	justify-content: flex-end;
+	height: 100%;
+`;
+
+export const GroupItemContainer = styled.div`
+	${GroupStyling}
+	flex-direction: row;
+	margin-bottom: 4px;
+
+	&:not(:hover) ${GroupItemButtons} {
+		display: none;
+	}
+
+	${/* sc-selector */ GroupIcon}::after {
+		border-color: ${({ theme }) => theme.palette.secondary.lightest};
+	}
+`;
+
+export const GroupCollectionAccordion = styled(Accordion)`
+	&& {
+		border: 0;
+		background-color: transparent;
+		margin-top: 9px;
+
+		&::after {
+			opacity: 1;
+		}
+
+		&:last-of-type::after {
+			border-bottom-left-radius: 5px;
+		}
+
+		.MuiAccordionSummary-root {
+			${GroupStyling}
+			flex-direction: row-reverse;
+			padding-left: 14px;
+		}
+
+		.MuiAccordionSummary-expandIconWrapper {
+			background-color: ${({ theme }) => theme.palette.secondary.main};
+			color: ${({ theme }) => theme.palette.primary.contrast};
+			border-radius: 50%;
+			width: 16px;
+			height: 16px;
+			display: flex;
+			align-items: center;
+			justify-content: center;
+			padding-top: 1px;
+			box-sizing: border-box;
+
+			svg {
+				width: 8px;
+				height: 6px;
+			}
+
+			&:not(.Mui-expanded) {
+				transform: rotate(270deg);
+			}
+		}
+
+		.MuiAccordionDetails-root {
+			padding: 9px 0 0 16px;
+			overflow: hidden;
+		}
+	}
+`;
+
+export const GroupTitle = styled.span`
```

Comment:
You might need to add some truncation styling here. Unless you can be sure that group/collection titles can't be that long
![image](https://user-images.githubusercontent.com/62481250/234328489-76514682-e99b-40c3-8da1-c4e552c99ab3.png)